Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -235,30 +202,23 @@ theorem wcovBy_succ (a : α) : a ⩿ succ a :=
 theorem covBy_succ_of_not_isMax (h : ¬IsMax a) : a ⋖ succ a :=
   (wcovBy_succ a).covBy_of_lt <| lt_succ_of_not_isMax h
 
-theorem lt_succ_iff_of_not_isMax (ha : ¬IsMax a) : b < succ a ↔ b ≤ a :=
-  ⟨le_of_lt_succ, fun h => h.trans_lt <| lt_succ_of_not_isMax ha⟩
+theorem lt_succ_of_not_isMax_of_le (ha : ¬IsMax a) : b ≤ a → b < succ a :=
```

Comment:
I would prefer the `_of_not_isMax` to always come last in the lemma names, for consistency. Maybe swap the hypotheses around so that it still matches the naming convention
```suggestion
theorem lt_succ_of_le_of_not_isMax (hab : b ≤ a) (ha : ¬IsMax a) : b < succ a :=
```

Reply:
I applied this, but it makes using it a bit awkward. Is there a way to keep the order while having `of_not_isMax` at the end?